pkg/agent/agent.go

```diff
@@ -130,6 +130,10 @@ func (a *Agent) Start() error {
 	if a.p0fRunner == nil {
 		return nil
 	}
+	// Additional check for nil pointer wrapped in interface
```

Comment:
Did you have issues with this ?

Reply:
I did. IIRC If you're not running p0f it will crash randomly due to the pointer. So I put that fix in to catch it better.